app/konnect/configure/runtime-manager/index.md

```diff
@@ -3,19 +3,27 @@ title: Runtime Manager Overview
 no_version: true
 ---
 
-The Runtime Manager is a {{site.konnect_saas}} functionality module
-that lets you catalogue, connect to, and monitor the status of all runtimes in
-one place.
+The [Runtime Manager](https://konnect.konghq.com/runtime-manager)
+is a {{site.konnect_saas}} functionality module
+that lets you catalogue, connect to, and monitor the status of all runtime
+groups and instances in one place, as well as manage group configuration.
 
-With {{site.konnect_short_name}} acting as the control plane, a runtime
+The Runtime Manager overview page displays a list of
+runtime groups currently owned by the organization. From here, you can add or
+delete runtime groups, or go into each individual group to manage runtime
+instances and their global configuration.
+
+_[screenshot of overview page]_
```

Comment:
Do we have a test environment somewhere where I could grab a useful screenshot?

Reply:
If what's available in https://cloud.konghq.tech is not sufficient. Then the next closet thing are the mocks:
https://www.figma.com/file/G4xvSqX30F2vxrJmesZuke/Runtime-Groups-(Helium)?node-id=1087%3A61521

Reply:
Let's wait for it to be in an environment where we can take a screenshot. Mocks will be misleading

Reply:
Pushing this out to post phase 1 so that we have a stable environment first. Not sure that all the labels etc are really correct at the moment.